pyproject.toml

```diff
@@ -12,7 +12,7 @@ joblib = "1.4.2"
 pydantic = ">=2.6.3"
 python = ">=3.9,<4.0"
 requests = ">=2.25.0,<3.0.0"
-tenacity = "8.2.3"
+tenacity = "9.0.0"
```

Comment:
should also bump the sdk version to "0.9.3" now (I think you had this incremented, but I went and updated one since)